x-pack/plugins/security/index.js

```diff
@@ -45,6 +46,12 @@ export const security = (kibana) => new kibana.Plugin({
         hostname: Joi.string().hostname(),
         port: Joi.number().integer().min(0).max(65535)
       }).default(),
+      authc: Joi.object({})
```

Comment:
__note:__ even though we drop it in 7.0 I wanted it to look similar to what we have in 7.2+.